app/encoding.go

```diff
@@ -1,11 +1,70 @@
 package app
 
 import (
-	"github.com/cosmos/cosmos-sdk/simapp/params"
-	evmenc "github.com/evmos/ethermint/encoding"
+	"github.com/cosmos/gogoproto/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+
+	"cosmossdk.io/simapp/params"
+	"cosmossdk.io/x/tx/signing"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/address"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/std"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/migrations/legacytx"
+	"github.com/cosmos/cosmos-sdk/x/auth/tx"
+
+	evmcryptocodec "github.com/evmos/os/crypto/codec"
+	"github.com/evmos/os/ethereum/eip712"
+	evmostypes "github.com/evmos/os/types"
+	erc20types "github.com/evmos/os/x/erc20/types"
+	evmtypes "github.com/evmos/os/x/evm/types"
+	ethcryptocodec "github.com/realiotech/realio-network/crypto/codec"
 )
 
 // MakeEncodingConfig creates the EncodingConfig for realio network
 func MakeEncodingConfig() params.EncodingConfig {
```

Comment:
This should change after removing legacy eip712

Reply:
sorry, can you elaborate that ?

Reply:
Sorry I made a mistake, confused the eip712 with the legacy eip712. This is no needed.